### DISCUSSION TITLE
PSM.h library file link

### DISCUSSION THREAD

**Discussion by rarenu** [Zer0-bit/gaggiuino#407] [Q&A] [answered]
Please share the link for the correct PSM.h library file to download. Thanks

**Comment by banoz** [comment]
https://github.com/Zer0-bit/gaggiuino/blob/3247a2a45fef00d9a940065d1603c88de6612514/platformio.ini#L119

**Comment by rarenu** [comment, marked as answer]
Hi, thank you for your immediate help.
With best regards.
